dashboard/public/js/main.js

```diff
@@ -308,19 +308,28 @@ function createGraph(type, element, route) {
 					data: response.data,
 					options: (response.options ? response.options : {})
 				});
-				myChart.options.onClick = function(e){
-					var activePoints = myChart.getElementsAtEvent(e);
-					// Avoid console erros when clicking on any white space in the chart
-					var index = activePoints.length ? activePoints[0]._index : -1;
-					if (index > -1 && response.graph === 'bar'){
-						window.location.href = `/dashboard/journal/${response.journalIDs[index]}?uid=${response.userIDs[index]}&type=private`;
-					}else if(index > -1 && response.graph === 'doughnut'){
```

Comment:
And this condition to:
```suggestion
					}else if(index > -1 && type === 'top-activities')
```